### ISSUE TITLE
Bot removes square brackets and text in those square brackets from song title when scrobbling

### ISSUE THREAD

**Issue by walllable** [fmbot-discord/fmbot#149]
**Command to reproduce & Last.FM username:**
.play https://open.spotify.com/track/35GYDmixAiE0d36qSPqfuO?si=537ce8f11df74e4f

**Describe the bug:**
The bot ignores square brackets in song titles, leading to incorrect scrobble titles.

**Screenshots:**
![image](https://user-images.githubusercontent.com/10066958/133231006-7bf5deec-9d7e-472a-81b7-1691daef91fc.png)

What it scrobbles: ![image](https://user-images.githubusercontent.com/10066958/133231863-54423091-bc48-4d1e-8786-9d5bec62ab2b.png)
What it should look like: 
![image](https://user-images.githubusercontent.com/10066958/133231819-89cc959a-4d05-42a3-a6f5-6db6905ec0b7.png)

**Version (use`.fmstatus`):**
![image](https://user-images.githubusercontent.com/10066958/133231090-ec30116f-a16a-4bce-9aa8-34f2fb994dfa.png)

**Comment by Whanos** [comment]
From looking at the code, I believe this is intentional - Why, I have no idea. The bot devs should confirm whether this is intended behavior or not first.

**Comment by th0mk** [comment]
This was indeed intentional; it was because some music bots had direct links to the song on Spotify or Youtube in the description.

For example, the description would have this: `[Mall Grab - Love Reigns](https://open.spotify.com/track/78Yqfmq11yBeibacqvl4Y4)`

If there's a Spotify link we just grabbed the song directly from Spotify, but in case of YouTube we only got the title that was in the first [] bracket. That's what caused the issue here.

Hydra doesn't seem to add links to the description (anymore?) so I've disabled this for Hydra scrobbling. Coming soon to a release near you, I'll update once its out :)
